python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_decorator.py

```diff
@@ -305,6 +308,39 @@ def my_dbt_assets(): ...
     assert my_dbt_assets.backfill_policy == expected_backfill_policy
 
 
+@pytest.mark.parametrize(
+    "retry_policy",
+    [
+        None,
+        RetryPolicy(max_retries=1),
+        RetryPolicy(max_retries=2, delay=1, jitter=Jitter.FULL),
+    ],
+    ids=[
+        "no retry policy",
+        "retry policy",
+        "retry policy with jitter",
+    ],
+)
+def test_retry_policy(
+    test_jaffle_shop_manifest: Dict[str, Any],
+    retry_policy: Optional[RetryPolicy],
+) -> None:
+    class CustomDagsterDbtTranslator(DagsterDbtTranslator):
+        def get_freshness_policy(self, _: Mapping[str, Any]) -> Optional[FreshnessPolicy]:
+            # Disable freshness policies when using static partitions
+            return None
+
+    @dbt_assets(
+        manifest=test_jaffle_shop_manifest,
+        retry_policy=retry_policy,
+        dagster_dbt_translator=CustomDagsterDbtTranslator(),
+    )
+    def my_dbt_assets(): ...
```

Comment:
This isn't needed, since we're not testing partitions.

```suggestion
    @dbt_assets(
        manifest=test_jaffle_shop_manifest,
        retry_policy=retry_policy,
    )
    def my_dbt_assets(): ...
```